src/codecs/html/README.md

```diff
@@ -38,3 +38,126 @@ Some of the main external dependencies:
 - `js-beautify` for pretty generated HTML.
 
 - `json5` for more human readable representation of arbitrary values.
+
+---
+
+## Microdata and semantic tag usage (WIP)
+
+### Schema nodes to add microdata to
+* Article
+    * `div` `itemtype='schema:Article'`
```

Comment:
What about `<article itemtype="schema:Article">`?

Reply:
Agreed on using `article` but this ties back to your overall comment:

> Perhaps a useful guideline would be to use itemtype on any schema:Thing, even if they have HTML equivalents. e.g. use <video itemtype="http://schema.org/VideoObject">. I think that for machine-readability, that is useful.

I would argue that always adding `itemtypes` might be redundant, and I'd prefer not to commit to things if we're not actively making use of them as such things easily gather dust and add to responsibilities.
The semantic tags themselves are already machine readable and the tags would be adding to the page download sizes.

Reply:
Fair point. I think we need to find out more about the "consumers" and what they want - in this case the web crawlers / other machines. If they semantically link a HTML `<article>` to a https://schema.org/Article, then yes it's redundant. If not, then it adding these `itemtype` provides that semantic link for them. Note that I am not proposing using `itemprop` on all nodes, only those derived from `Thing`, which will exclude many of the content nodes, like `h1`, `p` etc.